docs/internal-documentation/maintenance/ReleaseProcess-release.md

```diff
@@ -378,16 +409,20 @@ branch `merge-develop-into-master` is correct, and a pull request into
 [Changes GUI]:                             https://github.com/zonemaster/zonemaster-gui/blob/develop/Changes
 [Changes LDNS]:                            https://github.com/zonemaster/zonemaster-ldns/blob/develop/Changes
 [Changes Zonemaster]:                      https://github.com/zonemaster/zonemaster-gui/blob/develop/Changes
+[Create Docker Image]:                     ReleaseProcess-create-docker-image.md
+[Docker Hub]:                              https://hub.docker.com/u/zonemaster
 [Engine.pm]:                               https://github.com/zonemaster/zonemaster-engine/blob/develop/lib/Zonemaster/Engine.pm
 [Installation.md GUI]:                     https://github.com/zonemaster/zonemaster-gui/blob/develop/docs/Installation.md
 [LDNS.pm]:                                 https://github.com/zonemaster/zonemaster-ldns/blob/develop/lib/Zonemaster/LDNS.pm
 [NVM]:                                     https://github.com/nvm-sh/nvm
 [Node.js]:                                 https://nodejs.org/en/
 [PAUSE]:                                   https://pause.perl.org/
-[Versions and releases]:                   https://github.com/zonemaster/zonemaster/blob/develop/docs/design/Versions%20and%20Releases.md
+[Versions and releases2]:                   https://github.com/zonemaster/zonemaster/blob/develop/docs/design/Versions%20and%20Releases.md
```

Comment:
What's this for?

Reply:
Just some debug code I forgot to remove. Fixed.